packages/node/src/indexer/ds-processor.service.spec.ts

```diff
@@ -38,10 +39,11 @@ function getTestProject(
 describe('DsProcessorService', () => {
   let service: DsProcessorService;
   let project: SubqueryProject;
+  let nodeConfig: NodeConfig;
```

Comment:
Does nodeConfig need to be defined? Looks to be the same in other test files too